### PR TITLE
add units conversion rates

### PR DESCRIPTION
**Pull request by younies**
##### Checklist

- [ ] Issue filed: https://unicode-org.atlassian.net/browse/CLDR-_____
- [ ] Updated PR title and link in previous line to include Issue number

### CODE REVIEW

File: common/supplemental/units.xml [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+Copyright © 1991-2013 Unicode, Inc.
+CLDR data files are interpreted according to the LDML specification (http://unicode.org/reports/tr35/)
+For terms of use, see http://www.unicode.org/copyright.html
+-->
+
+<supplementalData>
+	<version number="$Revision$"/>
+	<convertUnits>
+		<convertUnit source='g-force' target='meter-per-second-squared' factor='9.80665' />
+		<convertUnit source='meter-per-second-squared' target='meter-per-second-squared' factor='1' />
```
Comment (sffc) [inline common/supplemental/units.xml]: Do you need to include the identity source/target entries?

File: common/supplemental/units.xml [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+		<convertUnit source='arc-minute' target='degree' factor='0.016666666666666666' />
```
Comment (sffc) [inline common/supplemental/units.xml]: Use a rational fraction

File: common/supplemental/units.xml [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+		<convertUnit source='arc-second' target='degree' factor='0.0002777777777777778' />
+		<convertUnit source='degree' target='degree' factor='1' />
+		<convertUnit source='radian' target='degree' factor='0.01745329' />
```
Comment (sffc) [inline common/supplemental/units.xml]: This is an example where the conversion is an irrational number.  Do we have a standard for how many significant figures we want in such cases?

File: common/supplemental/units.xml [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+		<convertUnit source='revolution' target='degree' factor='360' />
+		<convertUnit source='acre' target='square-foot' factor='43560' />
+		<convertUnit source='hectare' target='square-foot' factor='107639' />
```
Comment (sffc) [inline common/supplemental/units.xml]: hectare should go to square-meter

File: common/supplemental/units.xml [hunk 5]
```diff
@@ -0,0 +1,126 @@
...
+		<convertUnit source='square-centimeter' target='square-meter' factor='0.0001' />
+		<convertUnit source='square-foot' target='square-meter' factor='0.09290304' />
```
Comment (sffc) [inline common/supplemental/units.xml]: Don't need to include entries for the compound units; you need foot to meter, but not square foot to square meter (here and elsewhere)

File: common/supplemental/units.xml [hunk 6]
```diff
@@ -0,0 +1,126 @@
...
+		<convertUnit source='square-meter' target='square-foot' factor='100000000/9290304' />
+		<convertUnit source='square-inch' target='square-foot' factor='1/144' />
+		<convertUnit source='square-kilometer' target='square-meter' factor='1000000' />
+		<convertUnit source='square-mile' target='square-foot' factor='27878400' />
+		<convertUnit source='square-yard' target='square-foot' factor='9' />
+		<convertUnit source='dunam' target='square-meter' factor='1000' />
+		<convertUnit source='bit' target='bit' factor='1' />
+		<convertUnit source='byte' target='bit' factor='8' />
+		<convertUnit source='gigabit' target='bit' factor='1000000000' />
```
Comment (sffc) [inline common/supplemental/units.xml]: Don't need to include entries for the SI prefix units (here and elsewhere)

File: common/supplemental/units.xml [hunk 7]
```diff
@@ -0,0 +1,126 @@
...
+		<convertUnit source='light-year' target='meter' factor='9.46073e-15' />
+		<convertUnit source='micrometer' target='meter' factor='1e-06' />
+		<convertUnit source='mile' target='foot' factor='5280' />
+		<convertUnit source='millimeter' target='meter' factor='0.001' />
+		<convertUnit source='nanometer' target='meter' factor='1e-09' />
+		<convertUnit source='nautical-mile' target='meter' factor='1852' />
+		<convertUnit source='picometer' target='meter' factor='1e-12' />
+		<convertUnit source='point' target='meter' factor='35277.78' />
+		<convertUnit source='yard' target='foot' factor='3' />
+		<convertUnit source='solar-radius' target='meter' factor='695,510,000' />
```
Comment (sffc) [inline common/supplemental/units.xml]: Remove the grouping separators ?

**Comment by macchiati** [comment]
Closed, since the info will be merged into 292.

**Pull request closed without merging**